epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -603,6 +604,44 @@ private MethodSpec generateSetter(ClassToGenerateInfo helperClass, AttributeInfo
         .build();
   }
 
+  private MethodSpec generateReset(ClassToGenerateInfo helperClass) {
+    Builder builder = MethodSpec.methodBuilder("reset")
+        .addAnnotation(Override.class)
+        .addModifiers(Modifier.PUBLIC)
+        .returns(helperClass.getParameterizedGeneratedName())
+        .addStatement("super.reset()");
```

Comment:
I mentioned before that maybe we should have `super.reset() be called last, so that if a model want's a different default value (like true for a boolean) they could override `reset` and have that method be called last so that our default values don't override there's.

What do you think? I know its a pain to change the tests so I can do this is you want.